tests/kernel/sched/schedule_api/src/test_sched_timeslice_and_lock.c

```diff
@@ -142,7 +142,7 @@ void test_sleep_cooperative(void)
 
 void test_busy_wait_cooperative(void)
 {
-	/* set current thread to a preemptible priority */
+	/* set current thread to a cooperative priority */
```

Comment:
Those are synonyms[1].  "Preemptable" means "can be preempted". 

[1] Except for the misspelling.  Proper English construction spells this "preemptable" and not "preemptible", though both forms are common if you Google.  English spelling is crazy.

Reply:
? But cooperative threads are not supposed to be preempted, so isn't this right?

(There's only one use of `preemptable` in tree outside the spelling.txt map that prefers it as `preemptible.`  From k_work I've found preferring "canceling" over "cancelling", but always "cancelling" never "canceling", to be a it odd.)

Reply:
Oh, duh.  Never mind, reviewed code too early in the day. :)